packages/app-mobile/ios/Joplin/Info.plist

```diff
@@ -104,7 +104,7 @@
 		<string>UIInterfaceOrientationPortraitUpsideDown</string>
 	</array>
 	<key>UIUserInterfaceStyle</key>
-	<string>Light</string>
+	<string>Automatic</string>
```

Comment:
This change seems to be required for `react-native` to return any color theme that isn't light (see https://stackoverflow.com/a/71600340 and https://developer.apple.com/documentation/bundleresources/information_property_list/uiuserinterfacestyle).

It also seems to fix https://github.com/laurent22/joplin/issues/5775